_includes/mini-curriculum/debugging/debugger.md

```diff
@@ -151,13 +151,12 @@ debugging that you use Pry, but it will make life nicer.
 
 We're also going to use debugger (`gem install debugger`). This is a
 Ruby 1.9.3 gem, so make sure you have that version. You may also wish
-to grab my [.pryrc][pryrc] and [.rdebugrc][rdebugrc] files. Download
-them to your home (`~`) directory and rename them with an initial
+to grab my [.pryrc][pryrc] file. Download
```

Comment:
Move this link and corresponding instructions to the previous paragraph, since they're no longer relevant to the `debugger` gem.